tesla_api/climate.py

```diff
@@ -16,6 +16,12 @@ async def start_climate(self):
     async def stop_climate(self):
         return await self._vehicle._command("auto_conditioning_stop")
 
+    async def start_preconditioning_max(self):
+        return await self._api_client.post('vehicles/{}/command/set_preconditioning_max'.format(self._vehicle_id),{'on':True})
```

Comment:
Is there a reason this isn't implemented the same as the surrounding code?
i.e. `self._vehicle._command("set_preconditioning_max", {'on':True})`